### ISSUE TITLE
Manage all configuration through Configurator

### ISSUE THREAD

**Issue by tfausak** [tfausak/hairy#2]
http://hackage.haskell.org/package/configurator

It makes a lot of sense to me to put all the configuration in a file and then load that file instead of reading environment variables. There should still be one environment variable to control the environment (`SCOTTY_ENV`), but other stuff should go in the file. It can have defaults that can be overridden by environment-specific sections. For instance:

```
port = 3000
production { port = 80 }
```

At least these should be handled through the configuration file:

- Database (user, pass, host, port, database name, pool size)
- Port
- Verbose startup
- Static file caching

Basically anything in Scotty's `Options` or Wai's `Settings` seems like fair game.

**Comment by tfausak** [comment]
Although this would be nice to have, I don't think I'm going to spend any more time on this project. For an example of a similar project using Configurator, check out [Ledger](https://github.com/asm-products/ledger-backend).
